tests/test_build_js/source/code.js

```diff
@@ -242,6 +242,15 @@ class SimpleClass {
 
     /**
      * Static.
+     *
+     * @see nonStaticMethod
+     */
+    static staticMethod() {}
+
+    /**
+     * Non-static member.
+     *
+     * @see staticMethod
      */
-    static noUseOfThis() {}
```

Comment:
I changed the name so the tests are easier to read. `noUseOfThis` suggests to me I shouldn't be seeing it in test output, but it's intentionally there.